build.proj

```diff
@@ -663,6 +671,16 @@
     <Error Text="Binscope failed, please check the files in src/Package/BinScope"/>
   </Target>
 
+  <Target Name="TestFailureCheck">
+    <Exec Command="&quot;$(PowerShellCommand)&quot; -NonInteractive -NoLogo -NoProfile -Command &quot;$(MSBuildProjectDirectory)\tools\CheckTestFailures.ps1 -FailedTestFolder $(TestOutputDirectory)/FailingTests &quot;"
```

Comment:
@maddieclayton since the `CheckTestFailures.ps1` script is so small, we should be able to replace the script call with one line of PowerShell;

```powershell
Get-ChildItem $(TestOutputDirectory)/FailingTests | % { throw "Failing tests, please check files in src/TestResults/FailingTests" }
```